### PR TITLE
Implemented assign/unassign user to logical unit

### PR DESCRIPTION
**Pull request by Andu033**
issue #3

### CODE REVIEW

**Comment by RaduCStefanescu** [comment]
Hei @Andu033 
Thank you for your PR, but it seems the automated checks are failing, and also we merged a PR that changes the structure of documents.
Can you please have a look and make the required changes?
If you need additional info, please contact me or @mihairinzis on Slack

**Comment by Andu033** [comment]
@RaduCStefanescu I modified my PR and now the checks are passed. Also updated the way I implement the functionality. Sorry for late response.

**Comment by RaduCStefanescu** [comment]
@Andu033 thank you for the changes to the PR, I will have a look at them

**Comment by RaduCStefanescu** [comment]
Hey @Andu033 
Had a look at your PR. How about you try a different approach?
Create a new entity, let's call it Assignments.
The entity will contain the id of the user and the id of the document node, and it will also include a status, for example ACTIVE, COMPLETED or REVOKED.
Whenever there is a request for a new assignment, we just create a new object that links the user to the document node, in state ACTIVE. 
We should also let an admin user change the states of the assignment, through exposing operations in the controller.

File: src/main/java/com/code4ro/legalconsultation/service/impl/DocumentNodeServiceImpl.java [hunk 1]
```diff
@@ -34,4 +45,39 @@ public DocumentNode parse(String pdfContent) {
...
+
+    @Override
+    @Transactional
+    public List<DocumentNodeDto> assignUser(UUID userId, UUID nodeId){
+        User user = userRepository.findById(userId)
+                .orElseThrow(EntityNotFoundException::new);
+        DocumentNode documentNode = documentNodeRepository.findById(nodeId)
+                .orElseThrow(EntityNotFoundException::new);
+
+        user.getOwnedDocuments().add(documentNode);
```
Comment (mihaiba) [inline src/main/java/com/code4ro/legalconsultation/service/impl/DocumentNodeServiceImpl.java]: I suggest to add a method on the user model:
`user.addDocument(documentNode)` and make the `getOwnedDocuments` method to return a copy of the internal List

**Comment by Andu033** [comment]
@RaduCStefanescu I updated the PR with requested implementation.

**Comment by georgebejan** [comment]
#113 is the updated PR for this

**Comment by RaduCStefanescu** [comment]
This is replaced by #113

**Pull request closed without merging**